ui/Main.elm

```diff
@@ -59,35 +59,41 @@ update msg ({ filterValues } as model) =
 
         UpdateFilter newFilter ->
             let
+                noFilters =
+                    { model
+                        | productFilter = "all"
+                        , versionFilter = "all"
+                        , platformFilter = "all"
+                        , channelFilter = "all"
+                        , localeFilter = "all"
+                        , buildIdFilter = ""
```

Comment:
Side remark: it really feels like we should have a `Filters` type, and some `init` function for it. We could then have `FilterMsg`, an update function for it, and a dedicated module. 

Could be done in a next PR though.